### ISSUE TITLE
Add list and listitem role tags to the divs in the search result tables

### ISSUE THREAD

**Issue by nanderv** [Bellettrie/library_system#825]
These should be added to the mobile version (with the cards) for now, since tables suck for screenreaders. 

Add aria-hidden=true to the other table.
